ci/Dockerfile

```diff
@@ -5,8 +5,16 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt update && \
     apt install -y curl gnupg && \
     curl https://dist.apache.org/repos/dist/dev/arrow/KEYS | apt-key add - && \
-    printf "deb [arch=amd64] https://dl.bintray.com/apache/arrow/ubuntu/ bionic main\ndeb-src https://dl.bintray.com/apache/arrow/ubuntu/ bionic main\n" | tee /etc/apt/sources.list.d/apache-arrow.list
-
-RUN apt update -y && \
-    apt install -y --no-install-suggests ceph-mon ceph-osd apt-transport-https gnupg libarrow-dev libparquet-dev && \
+    printf "deb [arch=amd64] https://dl.bintray.com/apache/arrow/ubuntu/ bionic main\ndeb-src https://dl.bintray.com/apache/arrow/ubuntu/ bionic main\n" | tee /etc/apt/sources.list.d/apache-arrow.list && \
+    apt update && \
+    apt install -y --no-install-suggests \
+      ceph-mon \
+      ceph-osd \
+      apt-transport-https \
+      gnupg \
+      libarrow-dev \
+      libparquet-dev && \
     rm -rf /var/lib/apt/lists/* /var/tmp/*
+
+COPY build/bin/run-query build/bin/sky_tabular_flatflex_writer build/bin/ceph_test_skyhook_query /usr/bin/
+COPY build/lib/libcls_tabular.so.1.0.0 /usr/lib/libcls_tabular.so
```

Comment:
@jlefevre , the new pipeline is also implicitly ensuring that what we release is the minimum set of dependencies needed so that on a vanilla ceph installation, we only install the 3rdparty runtime requirements (libarrow and libparquet).

As you can see in this in lines 19-20, we only copy these binaries and the library. So could it be that the test is failing because there are other things that are needed? Where would that error messages be shown?

Reply:
and if those messages can be found somewhere, maybe we can add a command so that travis displays them when there's an error

Reply:
I don't think there are any other external dependencies than libarrow.  Boost is used but that is normally available in Ceph.  Here are the linked dependencies for[ run query](https://github.com/uccross/skyhookdm-ceph/blob/skyhook-luminous/src/progly/CMakeLists.txt#L20) and [cls_tabular](https://github.com/uccross/skyhookdm-ceph/blob/skyhook-luminous/src/cls/CMakeLists.txt#L274)

Reply:
Let me try on a blank cloudlab machine, maybe I missed something.

Reply:
oh, no worries, I asked in case you would know from the top of your head. I just checked inside that same container and all dependencies are available, so that's not the issue. I'm out of ideas then.

Reply:
I updated the cls COPY paths in the ci/Dockerfile and now the query tests pass.  But docker is failing immediately afterward now, if you can please take a look.